src/compile.coffee

```diff
@@ -184,25 +185,20 @@ compileHeaders = (httpHeaders) ->
   headers
 
 
-compileOrigin = (filename, parseResult, httpTransaction) ->
+compileOrigin = (filename, parseResult, mediaType, httpTransaction) ->
```

Comment:
I see `compileOrigin` as a descendant of `compile` function, so I would vote for same signature (i.e. `(mediaType, parseResult, filename)`).